web-registry/src/pages/Marketplace/Projects/Projects.types.ts

```diff
@@ -0,0 +1,12 @@
+import { SellOrderInfo } from '@regen-network/api/lib/generated/regen/ecocredit/marketplace/v1/query';
+
+import { ProjectCardProps } from 'web-components/lib/components/cards/ProjectCard';
+
+export interface ISellOrderInfo extends Omit<SellOrderInfo, 'id' | '$type'> {
```

Comment:
Minor nit: I might prefer we name this something like `UISellOrderInfo` - the `I` prefix is often used to correspond to a function the interface describes, where I believe here we’re creating the type to expliclty state that for UI purposes, it’ll always have an ID?

Reply:
I like that